docker-compose.yml

```diff
@@ -1,13 +1,15 @@
-app:
+twitarr:
```

Comment:
Tiny comment here: Docker's console logging always includes the container tag, so the longer the naming is, the more space you lose off the left side.

Reply:
People still use tiny console windows?

Reply:
Maybe I'm odd. Or just old. Or both. I still use 24 x 80 on a 27" display.

Reply:
(not the *whole* display, mind you)

Reply:
Well, it's traditional. I stretch mine across most of my screen 😁